.github/workflows/reuse.yml

```
@@ -0,0 +1,23 @@
+# ------------------------------------------------------------------------------------------------ #
+#                                This file is part of CosmoScout VR                                #
+# ------------------------------------------------------------------------------------------------ #
+
+# SPDX-FileCopyrightText: German Aerospace Center (DLR) <cosmoscout@dlr.de>
+# SPDX-License-Identifier: MIT
+
+name: Check Compliance with REUSE Specification
+
+on:
+  push:
+    branches:
+      - main
```

I think we can run this on all branches.

```suggestion
      - '**'
```